### ISSUE TITLE
nails: journal name recognition

### ISSUE THREAD

**Issue by janeyat** [aknutas/nails#6]
Hi! I am enjoying playing with HAMMER. Should I expect to get the same results as with your downloadable software? Or can I get more analysises with the downloadable software?

Many thanks,
Jane

**Comment by aknutas** [comment]
Hi,
   the online version is a server set up to run the software package available here. They are almost exactly the same and have the same output.

**Comment by janeyat** [comment]
So, could you look at an analysis I ran (http://hammer.nailsproject.net/analyses/264)?

Every article was from one journal, Anthrozoos, so of course the most popular publication is Anthrozoos. However, while I would not be surprised that Anthrozoos is the most highly cited journal in its own articles, I expect other titles to show up in Most Cited Publications. Is this a matter of granularity--there is no other cited journal that even comes close to how often Anthrozoos is cited in Anthrozoos?

For comparison I ran a topic search in WOS (http://hammer.nailsproject.net/analyses/265) and the list of highly cited journals looks as I expected.

**Comment by aknutas** [comment]
This could be a bug. Can you run a text search using a raw text editor in the source data file and search for other journal names? This way we can verify that other journal names are/are not present.

**Comment by janeyat** [comment]
Antti,

I’m too inept to do what you suggested, but I dumped the cited references from the first 500 articles into VantagePoint & pulled out the journal titles (and this was crudely done, not entirely accurate because of formatting inconsistencies in the data coming from WOS), but it’s more representative of what we should be seeing based on my experience with this field of research.

The top 10 cited journals look like:
Anthrozoos
Soc Anim
Appl Anim Behav Sci
Psychol Rep
J Pers Soc Psychol
J Am Vet Med Assoc
Anim Welfare
Psychol Bull
New Perspectives Our
J Soc Issues

Thanks,
Jane

**Comment by aknutas** [comment]
We should see those in the results. I am reopening this bug and we'll investigate this. Please send us the sample dataset. Thanks for reporting the issue!

(I am not sure about the schedule, though, but we'll try to get back to this eventually.)

**Comment by jsalminen** [comment]
Hi Jane,

Thank you for the feedback! Do you mean [this kind of chart](https://dl.dropboxusercontent.com/u/13579699/Screen%20Shot%202016-09-29%20at%2017.12.57%20.png) by the Most Cited Publications? I can't see your original analysis for some reason. 

If all data is from a single publication, only that publication will show up in this chart. The chart aggregates the citation counts of each article in your dataset by the publication where they were published. So, if we have a dataset like this: 

DocumentTitle       Publication     TimesCited
Article 1                  Science           3
Article 2                  Nature             9
Article 3                  Science           12

the chart will show

Science            15
Nature              9

At the moment the software does not calculate the most common publication names appearing in  article references. Would this be a useful feature for you?

Could you still send us the data you used in the analysis so we could ensure that everything is running as it should?

Best,
Juho

**Comment by janeyat** [comment]
Juho, try the attached file. Thanks!!!!!!

[Anthrozoos_2006-15.zip](https://github.com/aknutas/nails/files/501503/Anthrozoos_2006-15.zip)

**Comment by janeyat** [comment]
I ran the cited references from that file through VantagePoint to extract the journal titles and while this method was crude and not entirely accurate, the top ten Most Cited Journals are
The top 10 cited journals look like:
Anthrozoos
Soc Anim
Appl Anim Behav Sci
Psychol Rep
J Pers Soc Psychol
J Am Vet Med Assoc
Anim Welfare
Psychol Bull
New Perspectives Our
J Soc Issues

**Comment by jsalminen** [comment]
Hi Jane,

I run the analysis on your data, and it seems to be working correctly: [report](https://dl.dropboxusercontent.com/u/13579699/exploration.html). 

Most cited publications chart simply sums up the total number of citations (column TC in original data) for each article in the dataset by publication name (SO column). As there is only one publication, all citations are summed up, resulting in 2107 total citations. You can check this by opening the original data file in Excel and taking the sum of column TC.

If finding the most common publication names in article references is important, we could try adding that feature in the future. 

Antti, you can close the bug.

Best, 
Juho

**Comment by aknutas** [comment]
Many thanks, @jsalminen .

**Comment by janeyat** [comment]
Could you explain why the Most Cited Publications chart is created with this summing-up formula (or, it seems like cross multiplication, to me)? I don't understand it, and I don't feel that it presents an accurate picture of what's going on with my articles from Anthrozoos.

My data set has 517 articles (the "corpus"). I dumped all the cited references to these 517 articles into VantagePoint, which is able to extract the multiple citations from each field, and the number of cited references for this corpus is actually 14089, not 2107. Anthrozoos was cited 1554 times in 380 articles of the corpus. Clearly Anthrozoos cites Anthrozoos, but not only Anthrozoos.

However, VantagePoint is very difficult to use, and it's expensive, and my license runs out in a few months, so I have vested interest in finding an easy-to-use product (HAMMER) that could give me some of the results I'm looking for (e.g., most commonly cited journals).

In conclusion, yes, I think creating a feature "most common publication names appearing in cited references" is very important! Through that lens, we would see additional titles cited in Anthrozoos articles (as mentioned in one of my previous messages).

**Comment by jsalminen** [comment]
Hi,

Most cited publications chart identifies the most "prestigious" publications within the corpus using the citation counts from Web of Science. Essentially the chart answers the question "what publications within the corpus have been cited most often in Web of Science?" It is not very useful when all analysed articles are coming from the same publication: your corpus consists of one publication (Anthrozoos), and it is obviously cited most often elsewhere. Here is a [chart](https://dl.dropboxusercontent.com/u/13579699/cs%20most%20cited.png) from analysing all papers with topic "crowdsourcing". Publications such as Behavior Research Methods, CHI Conference, ACM, and MIS Quarterly seem to be authoritative when it comes to crowdsourcing.

To me it looks like you're trying to answer a different question, namely "what publications the articles in the corpus cite most often?" Article references are a bit messy, but I think we can take a look at it and try to add this feature to NAILS, too.

Best,
Juho

**Comment by janeyat** [comment]
So, is this where you are using Eigenvector to calculate relative influence of publications in the network? (Remember when, earlier, I said it seemed like a cross product rather than a sum?) I'll play around with this was some data sets that come from more than one journal.

There is a type of bibliometric analysis used especially in health sciences called "mapping the literature," and its methodology uses the number of times journals were cited in a corpus (the exact numbers, not sums or cross products) and other information to make collection development suggestions to librarians. So, I think offering this functionality (raw numbers of citations per journal title) would be useful to others besides little old me.

**Comment by jsalminen** [comment]
No, Eigenvector calculations mentioned in the CompSysTech paper refer to article citation network and they were done in [Gephi](https://gephi.org/). Here we simply sum up the total citations for each article according to publication where articles were published. I did the same calculation on Excel in one of your [files](https://dl.dropboxusercontent.com/u/13579699/Anthrozoos_2006-15.xlsx), see the bottom of the column named TC.

I'll take a look if this functionality could be included in NAILS, too. Thank you for the feedback, Jane!

Best,
Juho

**Comment by janeyat** [comment]
By the way, are you aware of this article that used HAMMER:

Shankar, RK, TH Shankarappa, TB Manjunatha Reddy, A Muniyappa. (2015) Scholarly Communication in the International Journal of Pet Management: A Bibliometric Analysis from 2005 to 2014. Journal of Agricultural & Food Information. 16:301-314.

**Comment by jsalminen** [comment]
Thanks, I wasn't aware of that! Nice to see people finding NAILS useful! :)
